node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always, { null: 'ignore' } ],
+    'no-else-return': [ warn, { allowElseIf: false } ],
+    'yoda': error,
+    'no-throw-literal': error,
+    'no-self-compare': error,
+    'no-return-assign': error,
+    'no-loop-func': error,
+    'valid-jsdoc': warn,
+    'for-direction': error,
+    'no-buffer-constructor': error,
+    'no-new-require': error,
+    'no-path-concat': error,
+    'comma-spacing': error,
+    'comma-style': error,
```

Comment:
Enforce a comma after the previous item, rather than before the next item
https://eslint.org/docs/rules/comma-style
```js
// Old
const obj = {
  a: 1
  , b: 2
}
// New
{
  a: 1,
  b: 2
}
```

Reply:
The advantage of the old system is less clutter on the git diffs. Saying that I almost always to the `new` option.

Reply:
@Eruant this comes up a lot. I don't buy the git diff thing at all, I get it, but one comma doesn't distract me so much that I have to adopt a new style of coding which reduces grokkability of the codebase :)